code/SitewideContentReport.php

```diff
@@ -235,6 +243,17 @@ public function getDataFieldValue($record, $fieldName)
             return $callback($record);
         }
 
+        // Links
+        if ($fieldName == "Link") {
+
+            return Controller::join_links(Director::absoluteBaseURL(), $record->Link());
```

Comment:
The problem is that there's a risk that baseurl is joined twice:

`Director::absoluteBaseURL` is the protocol + hostname + basepath
`$record->Link()` is basepath + relative url.


![image](https://cloud.githubusercontent.com/assets/936064/11022656/f01a81b0-86c8-11e5-992e-b5cd3d482d65.png)


Instead shall we use `$record->AbsoluteLink()` which is on both File and SiteTree?